extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_dropdownLink.php

```diff
@@ -0,0 +1,5 @@
+<a href="<?= Sanitizer::encodeAttribute( $model['href'] ); ?>" class="wds-global-navigation__dropdown-link"
```

Comment:
@kvas-damian what parameters do you mean exactly? Do you think we should add something here or in JS?